### PR TITLE
Updated openslide to 3.4.0

### PR DESCRIPTION
**Pull request by mvictoras**

### CODE REVIEW

File: Library/Formula/openslide.rb [hunk 1]
```diff
@@ -14,6 +14,8 @@ class Openslide < Formula
   depends_on 'glib'
   depends_on 'openjpeg'
   depends_on 'cairo'
+  depends_on 'gdk-pixbuf'
+  depends_on 'sqlite'
```
Comment (jacknagel) [inline Library/Formula/openslide.rb]: OS X ships with sqlite; is that sufficient?

File: Library/Formula/openslide.rb [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 class Openslide < Formula
   homepage 'http://openslide.org/'
-  url 'http://download.openslide.org/releases/openslide/openslide-3.3.3.tar.gz'
-  sha1 '2315f0daa5d963e6ba9f1e67517cee44f9deabe5'
+  url 'https://github.com/openslide/openslide/releases/download/v3.4.0/openslide-3.4.0.tar.gz'
+  sha1 'edd28a142f7801a354949c05e7f1b7dcb246db6b'
   revision 1
```
Comment (jacknagel) [inline Library/Formula/openslide.rb]: remove the revision on version updates

**Comment by jacknagel** [comment]
ping

**Comment by mvictoras** [comment]
I was a little busy, will take a look at this next week.

Victor

On Sat, Jan 31, 2015 at 4:48 PM, Jack Nagel <notifications@github.com>
wrote:

> ping
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/35788#issuecomment-72340870>.
>



-- 
*Victor Mateevitsi*
PhD Candidate - University of Illinois at Chicago <http://www.uic.edu/>
Research Assistant - Electronic Visualization Laboratory
<http://www.evl.uic.edu/>
mvictoras@gmail.com
www <http://www.vmateevitsi.com> | LinkedIn
<http://www.linkedin.com/pub/5/919/845> | Twitter
<http://twitter.com/mvictoras> | UIC <http://www.uic.edu> | EVL
<http://www.evl.uic.edu>

**Comment by mvictoras** [comment]
Ok done.

On Sat, Jan 31, 2015 at 5:53 PM, Victor Mateevitsi <mvictoras@gmail.com>
wrote:

> I was a little busy, will take a look at this next week.
>
> Victor
>
> On Sat, Jan 31, 2015 at 4:48 PM, Jack Nagel <notifications@github.com>
> wrote:
>
>> ping
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/Homebrew/homebrew/pull/35788#issuecomment-72340870>.
>>
>
>
>
> --
> *Victor Mateevitsi*
> PhD Candidate - University of Illinois at Chicago <http://www.uic.edu/>
> Research Assistant - Electronic Visualization Laboratory
> <http://www.evl.uic.edu/>
> mvictoras@gmail.com
> www <http://www.vmateevitsi.com> | LinkedIn
> <http://www.linkedin.com/pub/5/919/845> | Twitter
> <http://twitter.com/mvictoras> | UIC <http://www.uic.edu> | EVL
> <http://www.evl.uic.edu>
>



-- 
*Victor Mateevitsi*
PhD Candidate - University of Illinois at Chicago <http://www.uic.edu/>
Research Assistant - Electronic Visualization Laboratory
<http://www.evl.uic.edu/>
mvictoras@gmail.com
www <http://www.vmateevitsi.com> | LinkedIn
<http://www.linkedin.com/pub/5/919/845> | Twitter
<http://twitter.com/mvictoras> | UIC <http://www.uic.edu> | EVL
<http://www.evl.uic.edu>

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `openslide 3.4.0`.

**Pull request closed without merging**